plugin/web/extensions/sidebar/popups/mobileDropdown/main.tsx

```diff
@@ -6,7 +6,7 @@ import MobileDropdown from ".";
   const element = document.getElementById("root");
   if (element) {
     const root = ReactDOM.createRoot(element);
-    root.render(<MobileDropdown />);
+    root.render(<MobileDropdown isMobile />);
```

Comment:
Its a "Mobile" Dropdown, so you don't need to pass a isMobile prop here :P It'll only be used in the mobile layout

Reply:
This makes sense. But the question is: 
How and where from can I pass the `isMobile` prop to child elements?

Reply:
Done!